crates/bevy_pbr/src/render/pbr.wgsl

```diff
@@ -106,7 +106,6 @@ fn fragment(in: FragmentInput) -> @location(0) vec4<f32> {
 
 #ifdef TONEMAP_IN_SHADER
         output_color = tone_mapping(output_color);
-#endif
 #ifdef DEBAND_DITHER
```

Comment:
Should this nested `if` be indented? Not sure if that's a thing in shaders.

Reply:
I'm not familiar with all of the conventions of this shader preprocessor, but it appears to be based on the C preprocessor, in which it is conventional not to indent nested ifs.

Reply:
This should not be indented. Not something introduced in this pr, but I opted to fix it here so we don't forget.

Reply:
Ah I misinterpreted. There were two indentation things:
1. The question being asked: do we indent nested ifdefs (no, at least not currently)
2. The unnecessarily nested lines (ex: `output_color = tone_mapping(output_color)`)

I fixed (2)